### ISSUE TITLE
protocol relative urls.

### ISSUE THREAD

**Issue by chriseppstein** [sass/libsass#1407]
```scss
@import url(//fonts.googleapis.com/css?family=Montserrat);
```

Returns (via node-sass):

```
{
  "message": "malformed URL",
  "column": 9,
  "line": 1,
  "file": "stdin",
  "status": 1
}
```

The way ruby sass parses this is that `url(` is the left side of a quote and `)` is the right hand side. everything in-between is parsed as the contents of the string.

This may be a duplicate. ¯\\_(ツ)_/¯

**Comment by xzyfer** [comment]
@chriseppstein thanks for the report. This has been fixed and will be 3.3.

**Comment by xzyfer** [comment]
Duplicate https://github.com/sass/libsass/issues/1298
